iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/extensions/AbstractVersionableNetworkExtensionXmlSerializer.java

```diff
@@ -25,6 +25,8 @@
  */
 public abstract class AbstractVersionableNetworkExtensionXmlSerializer<T extends Extendable, E extends Extension<T>> implements ExtensionXmlSerializer<T, E> {
 
+    private static final String INCOMPATIBILITY_NETWORK_VERSION_MESSAGE = "IIDM-XML version of network (";
```

Comment:
Why create this variable?

Reply:
It is a string used 3 times (error's message), Sonar considers not creating a variable as a code smell.